src/h5web/visualizations/line/hooks.ts

```diff
@@ -30,19 +31,23 @@ export function useAxisDomains(): AxisDomains | undefined {
 export function useDataPoints(): Vector3[] | undefined {
   const { data } = useProps();
   const axisDomains = useAxisDomains();
-  const { size } = useThree();
+  const {
+    size: { width, height },
+  } = useThree();
```

Comment:
Not sure what you think, but I tend to keep this kind of nested destructuring on separate lines because the way Prettier does it is quite ugly :(

```
const { size } = useThree();
const { width, height } = size;
```